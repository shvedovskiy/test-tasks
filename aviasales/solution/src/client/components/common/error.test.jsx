/* eslint-disable no-undef */
import React from 'react';

import Error from './error';


describe('<Error />', () => {
  const props = {
    onRetry: jest.fn(),
    message: 'Some Error',
  };
  const error = shallow(<Error {...props} />);

  it('renders properly', () => {
    expect(error).toMatchSnapshot();
  });
});
