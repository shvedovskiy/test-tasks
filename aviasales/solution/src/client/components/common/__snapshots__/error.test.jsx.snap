// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> renders properly 1`] = `
<Message>
  <h2>
    <span>
      –ë–∏–ª–µ—Ç—ã –∑–∞–≥—Ä—É–∑–∏—Ç—å –Ω–µ —É–¥–∞–ª–æ—Å—å
    </span>
     
    <span
      role="img"
    >
      üò•
    </span>
     
    (
    Some Error
    )
  </h2>
  <error__RetryButton
    onClick={[MockFunction]}
  >
    –ü–æ–≤—Ç–æ—Ä–∏—Ç—å
  </error__RetryButton>
</Message>
`;
